app/modules/appusage/scripts/uninstall.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+rm -f "${MUNKIPATH}preflight.d/appusage"
+rm -f "${CACHEPATH}appusage.txt"
+
+# Stop crankd daemon and remove crankd and database
+# Commented out to keep crankd around as it may be used by other services or scripts
+
+#/bin/launchctl unload -w /Library/LaunchDaemons/com.googlecode.pymacadmin.crankd.plist
```

Comment:
Adding crankd as an external dependency would make shift the decision to remove crankd to munki (if the crankd installer is distributed with munki)